### ISSUE TITLE
Allow for each phase line to contain several comma-separated actions

### ISSUE THREAD

**Issue by markusrobertjonsson** [markusrobertjonsson/learningsimulator#93]
Allow a phase line to contain comma-separated actions. Each action can be
- a stimulus presentation
- a variable assignment (that may contain a pre-defined set of functions, like `choose`)
- a phase line label, saying where to go next (unconditionally)

Lines where an action is a phase line label do not need to contain any `|` character.

For example
```
@phase ...
L1    stimulus, x=choose(1,2,3), L2
L2    ...
```

Should the actions after a phase line label be executed?

The syntax
```
@phase ...
L1    stimulus, x=choose(1,2,3), L2    | L3
L2    ...
L3    ...
```
should either give an error message or just disregard `| L3`.

See also #94

**Comment by markusrobertjonsson** [comment]
New idea from Magnus on pi-day 2020:
Only allow actions in logical part. That way one can easily see which lines contain stimulus exposures and which do not. For example

```
L1  s   |   x=4, L2
L2      |   n=choose(1,2,3), pull:L1 |  L2
```

**Comment by markusrobertjonsson** [comment]
Merged with #94
